charts/seed-bootstrap/charts/hvpa/templates/hvpa-crd.yaml

```
@@ -0,0 +1,972 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: hvpas.autoscaling.k8s.io
```

I think we should rename the API group to not contain `k8s.io` as this is "reserved" by the Kubernetes core/community projects. 

```suggestion
  name: hvpas.autoscaling.gardener.cloud
```

---

Discussed this with @rfranzke offline, and decided to keep it as it is as we plan to move hvpa-controller to autoscaling.k8s.io group
Discussion with the community ongoing.